src/RenderLoop.cpp

```diff
@@ -164,16 +146,15 @@ void RenderLoop::KeyEvent(const SDL_KeyboardEvent& event, bool down)
         {
             bool aspectCorrectionEnabled = !projectm_get_aspect_correction(_projectMHandle);
             projectm_set_aspect_correction(_projectMHandle, aspectCorrectionEnabled);
-            projectm_set_toast_message(_projectMHandle, aspectCorrectionEnabled ? "Aspect Correction Enabled"
-                                                                                : "Aspect Correction Disabled");
+            SDL_LogInfo(SDL_LOG_CATEGORY_APPLICATION, aspectCorrectionEnabled ? "Aspect Correction Enabled" : "Aspect Correction Disabled");
```

Comment:
Please use the POCO logging macros, otherwise the application will output stuff in two different places.